core/arch/arm/plat-sam/main.c

```diff
@@ -351,3 +353,14 @@ void plat_primary_init_early(void)
 {
 	matrix_init();
 }
+
+void itr_core_handler(void)
+{
+	atmel_saic_it_handle();
+}
+
+void main_init_gic(void)
```

Comment:
Hmm, now it's clear that we could have picked a better name for this function. :-)
We should change it in a later PR.

Reply:
Yep :)